frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -91,32 +116,53 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
       <ModalBody>
         Adding capacity for <strong>{getName(ocsConfig)}</strong>, may increase your cloud expenses.
         <div className="ceph-add-capacity__modal">
-          <div className="ceph-add-capacity_sc-dropdown">
-            <OCSStorageClassDropdown onChange={setStorageClass} defaultClass={storageClass} />
+          <div
+            className={classNames(
+              'ceph-add-capacity_sc-dropdown',
+              storageClass?.provisioner === NO_PROVISIONER
+                ? ''
+                : 'ceph-add-capacity_sc-dropdown__margin',
+            )}
+          >
+            <OCSStorageClassDropdown onChange={handleStorageClass} />
           </div>
-          <label className="control-label" htmlFor="requestSize">
-            Raw Capacity
-            <FieldLevelHelp>{labelTooltip}</FieldLevelHelp>
-          </label>
-          <div className="ceph-add-capacity__form">
-            <input
-              className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
-              type="number"
-              name="requestSize"
-              value={osdSizeWithoutUnit}
-              required
-              disabled
-            />
-            <div className="ceph-add-capacity__input--info-text">
-              x 3 replicas = <strong>{provisionedCapacity} TiB</strong>
+          {storageClass?.provisioner === NO_PROVISIONER ? (
+            <div
+              className="ceph-add-capacity__current-capacity"
+              data-test-id="ceph-add-capacity-pvs-available-capacity"
+            >
+              <div className="text-secondary ceph-add-capacity__current-capacity--text">
+                <strong>Available capacity:</strong>
+              </div>
+              {`${pvsAvailableCapacity} / ${ocsConfig.spec.storageDeviceSets[0].replica} replicas`}
```

Comment:
May be you can consider moving this to a separate component, so that PVs will be fetched only in case of baremetal.
Also its required at two places.

Reply:
Ack. We can take this up in a follow up PR.